parent/pom.xml

```
@@ -98,10 +98,6 @@
     </mailingLists>
 
     <properties>
-        <sonatypeOssDistMgmtNexusUrl>https://jakarta.oss.sonatype.org/</sonatypeOssDistMgmtNexusUrl>
-        <sonatypeOssDistMgmtSnapshotsUrl>${sonatypeOssDistMgmtNexusUrl}content/repositories/snapshots/</sonatypeOssDistMgmtSnapshotsUrl>
-        <sonatypeOssDistMgmtStagingUrl>${sonatypeOssDistMgmtNexusUrl}content/repositories/staging/</sonatypeOssDistMgmtStagingUrl>
-        <sonatypeOssDistMgmtReleasesUrl>${sonatypeOssDistMgmtNexusUrl}service/local/staging/deploy/maven2/</sonatypeOssDistMgmtReleasesUrl>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

```suggestion
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
        <autoPublish>true</autoPublish>
```
suggestion: make autoPublish default to true

---

> Because no child project of this POM would have access to login and click publish from the "staging" repository

:confused: That does not seem to be true. Projects can have access to do that.